addons/website_blog/views/website_blog_templates.xml

```diff
@@ -230,23 +223,23 @@ list of filtered posts (by date or tag).
             </div>
         </section>
 
-        <section id="o_wblog_post_main" t-attf-class="container pt-4 pb-5 #{'anim' in request.params and 'o_wblog_post_main_transition'}">
+        <section id="o_wblog_post_main" t-attf-class="pb-5 #{'anim' in request.params and 'o_wblog_post_main_transition'}">
             <!-- Sidebar-enabled Layout -->
-            <div t-if="opt_blog_post_sidebar" t-attf-class="mx-auto #{opt_blog_post_readable and 'o_wblog_read_with_sidebar'}">
-                <div t-attf-class="d-flex flex-column flex-lg-row #{opt_blog_post_readable and 'justify-content-between'}">
-                    <div id="o_wblog_post_content" t-attf-class="#{opt_blog_post_readable and 'o_container_small mx-0 w-100 flex-shrink-0' or 'w-lg-75'}">
+            <div t-if="opt_blog_post_sidebar" t-attf-class="mx-auto">
+                <div t-attf-class="d-flex flex-column flex-lg-row justify-content-start">
+                    <div id="o_wblog_post_content" t-attf-class="col-lg-9 col-12 px-0 mx-0 w-100 flex-shrink-0">
                         <t t-call="website_blog.blog_post_content"/>
                     </div>
-                    <div id="o_wblog_post_sidebar_col" t-attf-class="pl-lg-5 #{not opt_blog_post_readable and 'flex-grow-1 w-lg-25'}">
+                    <div id="o_wblog_post_sidebar_col" t-attf-class="px-0 pt-4 align-self-lg-start align-self-center">
```

Comment:
When there is nothing dynamic anymore, it should not be `t-attf-` anymore... unless this is inherited by something?